src/controllers/dynakube/dynakube_controller.go

```diff
@@ -237,6 +237,7 @@ func (controller *DynakubeController) reconcileDynaKube(ctx context.Context, dkS
 
 		err = endpointSecretGenerator.GenerateForDynakube(ctx, dkState.Instance)
 		if dkState.Error(err) {
+			log.Error(err, "failed to generate data-ingest secret for dynakube")
```

Comment:
we don't `log.Error`, please provide an information inf log and return the error

Reply:
Actually we do. There are 6 other occurrences of`log.Error(err` in this file and 65 in the project.
This error is consistent with other if branches in current function.